components/MashCards/InitialDecisionCard.tsx

```diff
@@ -29,21 +29,21 @@ const InitialDecisionCard = ({ mashReferral }: Props): React.ReactElement => {
               <dd>
                 <Link href={mashReferral.referralDocumentURI}>
                   <a>
-                    {mashReferral.clients[0]}
+                    {mashReferral.clients[0] + ' '}
                     {mashReferral.clients.length > 1 &&
-                      ` + ${mashReferral.clients.length - 1} `}
+                      `+ ${mashReferral.clients.length - 1} `}
                     (referral)
                   </a>
                 </Link>
               </dd>
             </div>
             <div>
-              <dt>Screening decision</dt>
-              <dd>{mashReferral.screeningDecision}</dd>
+              <dt>Requested support</dt>
+              <dd>{mashReferral.requestedSupport}</dd>
             </div>
             <div>
-              <dt>Referral category</dt>
-              <dd>{mashReferral.referralCategory}</dd>
+              <dt></dt>
+              <dd></dd>
```

Comment:
I don't like this but I'm fine with it for now, currently all the cards use the same css and utilise a grid template expecting 4 items, adding an empty item maintains alignment (assign button being all the way to the right)